src/platform/monitoring/DowntimeNotification/components/Down.jsx

```diff
@@ -2,12 +2,8 @@ import React from 'react';
 
```

Comment:
removing the "p" tag because with it the va-alert will not render the text and icon vertically aligned as there is no header